integration-tests/pkg/mock_sensor/server.go

```diff
@@ -489,9 +489,33 @@ func (m *MockSensor) pushEndpoint(containerID string, endpoint *sensorAPI.Networ
 // translateAddress is a helper function for converting binary representations
 // of network addresses (in the signals) to usable forms for testing
 func (m *MockSensor) translateAddress(addr *sensorAPI.NetworkAddress) string {
+	address := utils.IPAddress{}
+	ipNetwork := utils.IPNetwork{}
+
+	addressData := addr.GetAddressData()
+	if len(addressData) > 0 {
+		address = utils.IPFromBytes(addressData)
+	} else {
+		// If there is no address data IpNetwork should be set and represent
+		// a CIDR block or external IP address.
```

Comment:
What happens if we IpNetwork is not set either? Should we panic and stop the test?

Reply:
No. The comment wasn't accurate for the local address both the address data and IpNetwork can be blank. I have corrected the comment.